libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt

```diff
@@ -94,8 +94,8 @@ fun asCheckMessageRequest(reqData: ReadableMap): CheckMessageRequest? {
     return null
 }
 
-fun asReverseSwapFeesRequest(reqData: ReadableMap): ReverseSwapFeesRequest? {
-    val sendAmountSats = reqData.getDouble("sendAmountSats")
+fun asReverseSwapFeesRequest(reqData: ReadableMap): ReverseSwapFeesRequest {
+    val sendAmountSats = if (hasNonNullKey(reqData, "sendAmountSats")) reqData.getDouble("sendAmountSats") else null
```

Comment:
Sorry, I didn't realise it should be a ULong
```
val sendAmountSats = if (hasNonNullKey(reqData, "sendAmountSats")) reqData.getDouble("sendAmountSats").toULong() else null
```
or maybe this is clearer
```
fun asReverseSwapFeesRequest(reqData: ReadableMap): ReverseSwapFeesRequest {
    if (hasNonNullKey(reqData, "sendAmountSats")) {
        val sendAmountSats = reqData.getDouble("sendAmountSats")
        return ReverseSwapFeesRequest(sendAmountSats.toULong())
    }
    
    return ReverseSwapFeesRequest(null)
}
```

Reply:
Done in 6771091833e2e23095f676ecb07d7dfe507683b4